### PR TITLE
chiaki: init at 1.0.3

### PR DESCRIPTION
**Pull request by delroth**
###### Motivation for this change

Chiaki is a new open source client for the PS4 remote play protocol.

Note: I put it in `pkgs/games`, it's not a perfect fit but other gaming-centric apps like `steam` are there already, so there's precedent (thought emulators are in `misc`... so there's also negative precedent).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
executable works
leaf package

File: pkgs/games/chiaki/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  nativeBuildInputs = [
+    cmake pkgconfig protobuf python3Packages.python python3Packages.protobuf
+  ];
+  buildInputs = [ ffmpeg libopus qtbase qtmultimedia qtsvg protobuf SDL2 ];
+
+  doCheck = true;
+
+  meta = with lib; {
```
Comment (aanderse) [inline pkgs/games/chiaki/default.nix]: Please include `platforms`.

File: pkgs/games/chiaki/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    homepage = https://github.com/thestr4ng3r/chiaki;
```
Comment (aanderse) [inline pkgs/games/chiaki/default.nix]: Please quote if you don't mind.

**Review by aanderse — CHANGES_REQUESTED** [review]
Just missing one attribute. Looks good otherwise. :+1:

**Review by aanderse — APPROVED** [review]

**Comment by aanderse** [comment]
Thanks @delroth and @jonringer :tada:

**Pull request merged**